pkg/dispers/dispers/metadata.go

```diff
@@ -0,0 +1,141 @@
+package dispers
+
+import (
+	"errors"
+	"os"
+	"time"
+
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/couchdb/mango"
+	"github.com/cozy/cozy-stack/pkg/prefixer"
+)
+
+// Only the Conductor should save metadata
+var prefix = prefixer.ConductorPrefixer
+
+// Metadata are written on the confuctor's database. The querier can read those Metadata to know his query's state
+type Metadata struct {
+	Start       time.Time `json:"start,omitempty"`
+	End         time.Time `json:"end,omitempty"`
+	Duration    string    `json:"duration,omitempty"`
+	Host        string    `json:"host,omitempty"`
+	Tags        []string  `json:"tags,omitempty"`
+	Outcome     bool      `json:"outcome,omitempty"`
+	Error       string    `json:"error,omitempty"`
+	Name        string    `json:"name,omitempty"`
+	Description string    `json:"description,omitempty"`
+	Output      string    `json:"output,omitempty"`
+}
+
+// NewMetadata returns a new Metadata object
+func NewMetadata(name string, description string, tags []string) Metadata {
+	now := time.Now()
+	host, _ := os.Hostname()
+
+	return Metadata{
+		Start:       now,
+		Description: description,
+		Name:        name,
+		Tags:        tags,
+		Host:        host,
+	}
+}
+
+// Close finish writting Metadata
+func (m *Metadata) Close(msg string, err error) error {
```

Comment:
I'm not fan of `Close` in this context. For me, close makes sense for closing a file, closing a program, etc. Maybe `EndQuery`? 
Also, I don't see the point of passing an error in params? If an error occured before, it should be handled when it occurs?

Reply:
Not fan of `EndQuery` either. 

I want a query to have plenty metadata (or logs if you thing I should have prefered that name). I want it to be openned and closed by CI in server disper.mycozy.cloud and to be pushed by the Conductor in server disper.perso.cloud. 

Maybe `Build` could do the job ! :D